src/reset-password/service.ts

```diff
@@ -17,22 +18,28 @@ export async function sendResetPasswordEmail(user: User) {
     }
 
     const transporter = nodemailer.createTransport(process.env.SMTP_URL);
-
+    const token = await createToken(user);
+    const emailTemplate =
+    render('emails/reset-password-email', {
```

Comment:
The indentation is weird for this. I think I would put `render` on the line above.